CMakeLists.txt

```diff
@@ -165,8 +192,8 @@ else()
 endif()
 
 if(${CMAKE_HOST_SYSTEM_NAME} MATCHES "Darwin")
-    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mmacosx-version-min=10.12 -DUSE_SCHEDULER_SCALING_PTHREADS")
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++ -mmacosx-version-min=10.12")
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -mmacosx-version-min=11.0.0 -DUSE_SCHEDULER_SCALING_PTHREADS")
```

Comment:
I think that doing 10.14 would be better. I believe that if we do 11, then it wont work on less than Big Sur. We had previously sad we would keep inline with Homebrew and do at least the last 3 versions of MacOS.

Reply:
I believe this is needed for the M1, but I could be mistaken. Perhaps we need to double check.

Reply:
I can double-check this.

Reply:
Confirmed that `-mmacosx-version-min=10.14` works okay on M1, or at least as okay as `-mmacosx-version-min=11.0.0` does.